electrification.mss

```diff
@@ -209,7 +209,7 @@
        line-color: #97FF2F;
     }
 
-    [frequency=null][frequency!=0][voltage>=15000][voltage<25000] {
+    [frequency!=null][frequency!=0][voltage>=15000][voltage<25000] {
```

Comment:
This always overrules the two selectors after this. Should be `[frequency>0][voltage>=15000]` instead, so that it can be overridden by any subsequent matching selector.